cleaned_data/modsbyPID/rftaart:64.xml

```diff
@@ -0,0 +1,71 @@
+   <name authority="naf"
+         valueURI="http://id.loc.gov/authorities/names/no2003045282">
+      <namePart>Ramsey, Marshall</namePart>
+      <role>
+         <roleTerm authority="marcrelator"
+                   valueURI="http://id.loc.gov/vocabulary/relators/art">Artist</roleTerm>
+      </role>
+   </name>
+   <name>
+      <namePart>Melchor, Susan</namePart>
+      <role>
+         <roleTerm authority="marcrelator"
+                   valueURI="http://id.loc.gov/vocabulary/relators/ive">Interviewee</roleTerm>
+      </role>
+   </name>
+   <physicalDescription>
+      <form authority="aat" valueURI="http://vocab.getty.edu/aat/300033973">drawings (visual works)</form>
+   </physicalDescription>
+   <subject authority="lcsh"
+            valueURI="http://id.loc.gov/authorities/subjects/sh85146723">
+      <topic>Wildfires</topic>
+   </subject>
+   <subject authority="lcsh"
+            valueURI="http://id.loc.gov/vocabulary/graphicMaterials/tgm003678">
+      <topic>Evacuations</topic>
+   </subject>
+   <subject authority="lcsh"
+            valueURI="http://id.loc.gov/vocabulary/graphicMaterials/tgm002379">
+      <topic>Community service</topic>
+   </subject>
+   <subject authority="lcsh"
+            valueURI="http://id.loc.gov/vocabulary/graphicMaterials/tgm003413">
+      <topic>Dwellings</topic>
+   </subject>
+   <subject authority="lcsh"
+            valueURI="http://id.loc.gov/vocabulary/graphicMaterials/tgm010220">
+      <topic>Stoves</topic>
+   </subject>
+   <typeOfResource>text</typeOfResource>
+   <relatedItem displayLabel="Project" type="host">
+      <titleInfo>
+         <title>Rising from the Ashes: The Chimney Tops 2 Wildfires in Memory and Art.</title>
+      </titleInfo>
+   </relatedItem>
+   <relatedItem type="references">
+      <location>
+         <url>Waiting on interview transcript</url>
```

Comment:
@logantillman - Here is another item that we still need the link for. @markpbaggett - can you comment on this PR with any links that do become available once you add the captions/content?